Packs/Algosec/Scripts/AlgosecCreateTicket/AlgosecCreateTicket.py

```diff
@@ -0,0 +1,34 @@
+import demistomock as demisto  # noqa: F401
+from CommonServerPython import *  # noqa: F401
+
+
+def algosec_create_ticket():
```

Comment:
This can be in the `main` method

Reply:
added the method out of main for testing